### DISCUSSION TITLE
How can I enable the Dark Reader extension on a specific element of the page instead of the entire page?

### DISCUSSION THREAD

**Discussion by GideonBear** [darkreader/darkreader#13040] [Q&A]
My use case is for Proton Mail; they have a dark mode, so I would preferably not use dark reader for their websites (the theme also syncs to drive, calendar, etc.). But the emails themselves are still white. Before I used Gmail on light mode with Dark Reader, which worked perfectly.
If I could just enable dark reader on the `message-content` element, I could still use the native dark mode of Proton Mail, but not be blinded by the emails themselves.
I know there's the dev tools, can I perhaps use that?

**Comment by kanasimi** [comment]
#7448

**Reply by GideonBear** [reply]
#7448 is about enabling it only on subpaths of a website, my question is about enabling it only on certain **DOM elements** within a single page.

**Comment by Vyom-Agrawal** [comment]
There's no built-in way to target only the email body or specific element, so we can't just have the message content darkened while keeping the rest of Proton Mail in its native dark mode. Temporary workarounds like dev tools or manual CSS can invert single elements but aren't persistent/practical for daily use. There doesn't seem to be any other reliable way even after looking up from several sources. I guess we'll have to wait from the Proton mail update to fix this issue!

**Reply by GideonBear** [reply]
> I guess we'll have to wait from the Proton mail update to fix this issue!

What issue? What is Proton Mail supposed to do with this?
